src/main/java/hw10/FileLogger.java

```diff
@@ -0,0 +1,79 @@
+package hw10;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Objects;
+
+public class FileLogger {
+    private static final DateTimeFormatter fileDateFormatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+    private static final FileLoggerConfiguration configuration =
+            new FileLoggerConfigurationLoader()
+                    .loadConfiguration("C:/hillel/hillel_homework/src/main/java/hw10/configuration.txt");
```

Comment:
Please get rid of the absolute path - `"C:/hillel/hillel_homework/src/main/java/hw10/configuration.txt"`, and use the relative path instead. You can put your config in the `resources` folder and read it from there.